### DISCUSSION TITLE
What to do with training data if model has been reject?

### DISCUSSION THREAD

**Discussion by adambelniak** [suragnair/alpha-zero-general#178] [Q&A]
What do you thing about removing last training history if model has been rejected ?

**Comment by NMO13** [comment]
I don't see any reason why doing so. If the model has been rejected doesn't mean that the most recent training data is bad. Its more the opposite: It means that the training data is a valuable source for getting the agent smarter.

**Comment by adambelniak** [comment]
That's clear. I'm wondering, how we indicate that collected moves, during self-play phase, are bad?
